tests/iostability/test_iostability_degraded_path.py

```diff
@@ -220,3 +219,169 @@ def test_disk_near_full_read_in_degraded_s3bench(self):
         self.test_completed = True
         self.log.info("ENDED: Perform disk storage near full once and read in loop for %s days",
                       self.duration_in_days)
+
+    @pytest.mark.lc
+    @pytest.mark.io_stability
+    @pytest.mark.tags("TEST-40174")
+    def test_degraded_iteration_write_read_partial_delete(self):
```

Comment:
This test is very long. Can you divide into more reusable functions and use them in similar tests in this module.

Reply:
Added two function in lib.
perform_write_to_fill_system_percent and delete_workload